BAKKA_Editor/UserSettings.cs

```diff
@@ -62,4 +63,9 @@ public class SoundSettings
     public string HitsoundPath { get; set; } = "";
     public int HitsoundVolume { get; set; } = 50;
     public int HitsoundAdditionalOffsetMs { get; set; } = 0;
+}
+
+public class OptionsSettings
```

Comment:
We have to be careful when adding these new things. I'm not sure about the name of `OptionsSettings` - we should revisit this.

Reply:
Agreed on naming. I was following the existing naming conventions for the other menu bar items. Will move it anyways into the settings UI as you suggested.

Reply:
mmm, I see. The new settings in the dialog still live under UserSettings. In that case, I'll rename `OptionsSettings` to `CursorSettings`.